examples/SignData.purs

```diff
@@ -0,0 +1,45 @@
+module Ctl.Examples.SignData (main, example, contract) where
+
+import Contract.Prelude
+
+import Contract.Config (ConfigParams, testnetNamiConfig)
+import Contract.Log (logInfo')
+import Contract.Monad (Contract, launchAff_, liftedM, runContract)
+import Contract.Prim.ByteArray (RawBytes, rawBytesFromAscii)
+import Contract.Wallet (getChangeAddress, getRewardAddresses, signData)
+import Ctl.Internal.Serialization.Address (Address)
```

Comment:
It's concerning we have to import an internal type, to be able to write the type for a Contract exported function.